### ISSUE TITLE
Feature request: Keep rounded corners after hide

### ISSUE THREAD

**Issue by Microwave-WYB** [mlutfy/hidetopbar#345]
Is it possible to keep the top rounded corners when the panel is hidden?

**Comment by tuxor1337** [comment]
Yes, we can add an option for that. I will be happy to merge a PR about this. This will require the following line to be omitted if the setting is active:
https://github.com/mlutfy/hidetopbar/blob/1d3dd330bbea72e11671ded637df8837a96629bd/panelVisibilityManager.js#L106

**Comment by Microwave-WYB** [comment]
I want to try implement this feature as my first contribution.
I'm trying to add support for keeping rounded corners. I implemented `Settings.ui` and `Settings-40.ui` and I could add the switch successfully. However, I then implemented `prefs.js` and `panelVisibilityManager.js` to make it check for a key `keep-round-corners` in `self._settings`, but I got this error when opening the settings window from gnome extensions:
```
Error: GSettings key keep-round-corners not found in schema org.gnome.shell.extensions.hidetopbar
```
Then I tried to add this key to `schemas`, I still get the same error, and this folder is in `.gitignore`
What is a proper way to add this key?
